obspy/signal/spectral_estimation.py

```diff
@@ -1737,6 +1821,13 @@ def plot(self, filename=None, show_coverage=True, show_histogram=True,
                 selected.
         :type show_noise_models: bool, optional
         :param show_noise_models: Enable/disable plotting of noise models.
+        :type show_earthquakes: bool, optional
+        :param show_earthquakes: Enable/disable plotting of earthquake models like
+                in [ClintonHeaton2002]_ and [CauzziClinton2013]_. Defaults to 
```

Comment:
I think you forgot to add the bibtex item for the second publication?

Reply:
I tried ! But failed. Not this last time I believe :)